courses/utils_test.py

```diff
@@ -22,6 +22,10 @@
     generate_program_certificate,
     process_course_run_grade_certificate,
     sync_course_runs,
+    ensure_course_run_grade,
+    log,
```

Comment:
Please remove this import. The patch below isn't necessary, and if it was necessary it can be accomplished without the import